core/tests/snapshots.rs

```diff
@@ -218,7 +218,8 @@ fn run_bank_forks_snapshot_n<F>(
         accounts_package_sender,
     };
     for slot in 1..=last_slot {
-        let mut bank = Bank::new_from_parent(&bank_forks[slot - 1], &Pubkey::default(), slot);
+        let mut bank =
+            Bank::new_from_parent(bank_forks.get(slot - 1).unwrap(), &Pubkey::default(), slot);
```

Comment:
We were previously using `Index` here to get `slot - 1`, which would panic if that key wasn't present.
This simply makes it more obvious that there's a "potential" panic here - though I believe this is safe?

Reply:
I think I changed this since `bank_forks[x]` returns an `&Arc<Bank>` whereas `bank_forks.get(x)` does the clone internally. I'll change it back